Лаба 4/task 2.py

```diff
@@ -0,0 +1,20 @@
+import json
+
+
+def task() -> float:
+
+    data = [
+        {"score": 0.0009456152645028281, "weight": 1},
+        {"score": 0.003, "weight": 2},
+        {"score": 0.005, "weight": 3},
+        {"score": 2.274, "weight": 1}
+    ]
```

Comment:
Данные должны браться из файла
```suggestion
    filename = "input.json"
```